### PR TITLE
Bug fix/enviroment changes

### PR DESCRIPTION
**Pull request by ravirk91**
Thank you for your contribution.
Before submitting this PR, please make sure:
- [ ] PR description and commit message should describe the changes done in this PR
- [ ] Verify the PR is pointing to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [ ] Latest Code from master or specific release branch is merged to your branch
- [ ] No unwanted\commented\junk code is included
- [ ] No new warning upon build solution
- [ ] Code Summary\Comments are added to my code which explains what my code is doing
- [ ] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [ ] Sanity Tests are successfully executed for New and Existing Functionality
- [ ] Verify that changes are compatible with all relevant browsers and platforms.
- [ ] After creating pull request there should not be any conflicts
- [ ] Resolve codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [ ] Resolve code review comments
- [ ] Help Libray document is updated accordingly for Feature changes.

### CODE REVIEW

File: Ginger/GingerCore/GeneralLib/General.cs [hunk 1]
```diff
@@ -1319,6 +1320,16 @@ public static bool IsAdmin()
             return false;
 
         }
+
+        public static ObservableList<SolutionCategoryValue> GetSolutionReleaseValues()
```
Comment (MeniKadosh1) [inline Ginger/GingerCore/GeneralLib/General.cs:1324]: i don't General class the proper place for this, need to add it to Workspace.cs or to SolutionOperations classes

File: Ginger/Ginger/Environments/EnvsListPage.xaml.cs [hunk 2]
```diff
@@ -61,7 +63,10 @@ private void SetGridView()
             view.GridColsView = new ObservableList<GridColView>();
 
             view.GridColsView.Add(new GridColView() { Field = ProjEnvironment.Fields.Name, WidthWeight = 200 });
-            view.GridColsView.Add(new GridColView() { Field = ProjEnvironment.Fields.ReleaseVersion, WidthWeight = 80, Header = "Release Version" });
+
```
Comment (MeniKadosh1) [inline Ginger/Ginger/Environments/EnvsListPage.xaml.cs:66]: this is the page in which all is readOnly?

**Review by MeniKadosh1 — CHANGES_REQUESTED** [review]

File: Ginger/GingerCore/GeneralLib/General.cs (hunk 1 above)
Comment (ravirk91) [reply Ginger/GingerCore/GeneralLib/General.cs]: moved in SolutionOperations  class

File: Ginger/Ginger/Environments/EnvsListPage.xaml.cs (hunk 2 above)
Comment (ravirk91) [reply Ginger/Ginger/Environments/EnvsListPage.xaml.cs:65]: yes this is read only page

File: Ginger/GingerCoreNET/WorkSpaceLib/SolutionOperations.cs [hunk 3]
```diff
@@ -344,5 +346,19 @@ public string ConvertSolutionRelativePath(string relativePath)
             return OperatingSystemBase.CurrentOperatingSystem.AdjustFilePath(relativePath);
         }
 
+        public static ObservableList<SolutionCategoryValue> GetSolutionReleaseValues()
+        {
```
Comment (MeniKadosh1) [inline Ginger/GingerCoreNET/WorkSpaceLib/SolutionOperations.cs:350]: i still see the method on General class

**Review by MeniKadosh1 — APPROVED** [review]

**Pull request merged**